app/helpers/ama_layout_content_helper.rb

```diff
@@ -53,4 +53,8 @@ def notice(notice)
   def alert(alert)
     render "ama_layout/alert" if alert
   end
+
+  def success_message(success)
+    render "ama_layout/success" if success
```

Comment:
What do you think of passing the success message to the partial as a variable, so it doesn't haven't to read from the `flash` object?

`render partial: "ama_layout/success", locals: { message: success } if success`